tests/hats_import/margin_cache/test_margin_cache.py

```diff
@@ -38,13 +37,13 @@ def test_margin_cache_gen(small_sky_source_catalog, tmp_path, dask_client):
 
     assert len(data) == 13
 
-    assert all(data[PartitionInfo.METADATA_ORDER_COLUMN_NAME] == norder)
-    assert all(data[PartitionInfo.METADATA_PIXEL_COLUMN_NAME] == npix)
-    assert all(data[PartitionInfo.METADATA_DIR_COLUMN_NAME] == int(npix / 10000) * 10000)
+    assert all(data[paths.PARTITION_ORDER] == norder)
+    assert all(data[paths.PARTITION_PIXEL] == npix)
+    assert all(data[paths.PARTITION_DIR] == int(npix / 10000) * 10000)
```

Comment:
```suggestion
    assert all(data[paths.PARTITION_DIR] == int(npix / 10_000) * 10_000)
```
Inspired by something I saw recently 😃

Reply:
Updated here and another place.